web/client/components/manager/users/UserDialog.jsx

```diff
@@ -223,7 +223,7 @@ class UserDialog extends React.Component {
     };
 
     render() {
-        return (<Dialog onClickOut={this.props.onClose} modal draggable={false} maskLoading={this.props.user && (this.props.user.status === "loading" || this.props.user.status === "saving")} id="mapstore-user-dialog" className="user-edit-dialog" style={assign({}, this.props.style, {display: this.props.show ? "block" : "none"})}>
```

Comment:
@kasongoyo can you provide motivation on why you think this should not be tested?

@offtherailz please follow this post

Reply:
@MV88 I think the hide/show behavior is full managed by Dialog component, here I was working on UserDialog build on top of Dialog and what I did was just removing the onClickOut prop. If we need to test hide/show behaviour, then we can do that on Dialog component, perhaps on a separate issue than this one.

Reply:
@kasongoyo I understand your point. 

The clickOut is already tested inside the Dialog component and I see you are removing the functionality, so in some way it's a feature removal, so testing that the feature is not there anymore is strange (as well as we will have also manual tests for them). 

I don't want to loose too much time in an issue like this (that for me is not even a bug :smile: ) 

The problem is that we want the bug fixes has at least some unit tests, that prevent in any case the other developers from introduce a regression. 

Long story short, if it too hard to provide a test for this, we can evaluate to skip. If it's easy, please add

Reply:
@offtherailz thanks for detailed information. I have no problem adding tests until when it's hard to do so. The changes introduced are very less prone to regression because it's only affect single independent component so we can skip this.